lib/main.coffee

```diff
@@ -19,6 +19,11 @@ module.exports =
 
   provideLinter: ->
     helpers = require('atom-linter')
+    _path = require('path')
+    _interopRequireDefault = (obj) ->
```

Comment:
This looks like leftovers from being transpiled to ES5, can you explain why this is necessary here?